tests/unit/test_charm.py

```diff
@@ -175,15 +219,15 @@ def test_update_container_config(harness: Harness, mocked_sql_migration: MagicMo
 
 def test_on_config_changed_without_service(harness: Harness) -> None:
     setup_postgres_relation(harness)
-    harness.update_config({"login_ui_url": "http://some-url"})
+    setup_login_ui_relation(harness)
```

Comment:
Let's keep the unit tests as minimal as possible. Since the login ui relation is not mandatory for hydra to start (because you set up default urls), I'd simply remove this line
```suggestion

```